src/main/webapp/script.js

```diff
@@ -26,6 +26,11 @@ const mobileMenu = () => {
 //toggle the div of hamburger menu
 menu.addEventListener('click', mobileMenu);
 
+
+  // Add it to the page.
+  const greetingContainer = document.getElementById('greeting-container');
+  greetingContainer.innerText = greeting;
```

Comment:
We should delete this piece of code, seems kinda random

Reply:
Yeah couldn't remember why it was there, it doesn't do anything it was from a copy and past I did, I will delete it.

Reply:
@SantiM1 I actually I don't know if I was the one to put it seem to be in the code that Eric wrote so not sure, I can comment it for now and ask him later.